proto/translate/v1/translate.proto

```diff
@@ -83,6 +83,7 @@ message UpdateTranslationRequest {
   string service_id = 1;
   bool populate_translations = 2;
   Translation translation = 3;
+  google.protobuf.FieldMask update_mask = 4;
 }
```

Comment:
Let's change the order of fields in the proto message to make it more logical order.

PS. This cannot be done when the code is in the production.

```suggestion
  string service_id = 1;
  Translation translation = 2;
  google.protobuf.FieldMask update_mask = 3;
  bool populate_translations = 4;
}
```